content/docs/2.0/migration.md

```diff
@@ -114,6 +120,38 @@ spec:                                     #  <--- spec.scaleType are not needed
   # {list of triggers to create jobs}
 ```
 
+### Improved flexibility & usability of trigger metadata
+
+We've introduced more options to configure trigger metadata to give users more flexibility.
+
+> 💡 **NOTE:** Changes only apply to trigger metadata and don't impact usage of `TriggerAuthentication`
+
+Here's an overview:
+
+| Scaler | 1.x | 2.0 |
+|--------|--------|--------|
+| `azure-blob` | `connection` (**Default**: `AzureWebJobsStorage`) | `connectionFromEnv` |
+| `azure-monitor` | `activeDirectoryClientId` <br /> `activeDirectoryClientPassword` | `activeDirectoryClientId` <br /> `activeDirectoryClientIdFromEnv` <br /> `activeDirectoryClientPasswordFromEnv` |
+| `azure-queue` | `connection` (**Default**: AzureWebJobsStorage) | `connectionFromEnv` |
+| `azure-servicebus` | `connection` | `connectionFromEnv` |
+| `azure-eventhub` | `storageConnection` (**Default**: `AzureWebJobsStorage`) <br /> `connection` (**Default**: `EventHub`) | `storageConnectionFromEnv` <br /> `connectionFromEnv` |
+| `aws-cloudwatch` | `awsAccessKeyID` (**Default**: `AWS_ACCESS_KEY_ID`) <br /> `awsSecretAccessKey` (**Default**: `AWS_SECRET_ACCESS_KEY`) | `awsAccessKeyID` <br /> `awsAccessKeyIDFromEnv` <br /> `awsSecretAccessKeyFromEnv` |
+| `aws-kinesis-stream` | `awsAccessKeyID` (**Default**: `AWS_ACCESS_KEY_ID`) <br /> `awsSecretAccessKey` (**Default**: `AWS_SECRET_ACCESS_KEY`) | `awsAccessKeyID` <br /> `awsAccessKeyIDFromEnv` <br /> `awsSecretAccessKeyFromEnv` |
+| `aws-sqs-queue` | `awsAccessKeyID` (**Default**: `AWS_ACCESS_KEY_ID`) <br /> `awsSecretAccessKey` (**Default**: `AWS_SECRET_ACCESS_KEY`) | `awsAccessKeyID` <br /> `awsAccessKeyIDFromEnv` <br /> `awsSecretAccessKeyFromEnv` |
+| `kafka` | _(none)_ | _(none)_
+| `rabbitmq` | `apiHost` <br /> `host` | ~~`apiHost`~~ <br /> `host` <br /> `hostFromEnv` |
+| `prometheus` | _(none)_ | _(none)_ |
+| `cron` | _(none)_ | _(none)_ |
+| `redis` | `address` <br /> `host` <br /> `port` <br /> `password` | `address` <br /> `addressFromEnv` <br /> `host` <br /> `hostFromEnv` <br /> ~~`port`~~ <br /> `passwordFromEnv`
+| `redis-streams` | `address` <br /> `host` <br /> `port` <br /> `password` |  `address` <br /> `addressFromEnv` <br /> `host` <br /> `hostFromEnv` <br /> ~~`port`~~ <br /> `passwordFromEnv`
```

Comment:
This one I'm not sure about, can you verify please @zroubalik? https://github.com/kedacore/keda/pull/1072/files